app/models/gems_adoption.rb

```diff
@@ -0,0 +1,3 @@
+class GemsAdoption < ActiveRecord::Base
+	belongs_to :user
+end
```

Comment:
It's probably a good idea to validate that `gem`, `user` and `description` are not `nil` at the model level, what do you think?